components/warn_metric_ack_modal/warn_metric_ack_modal.tsx

```diff
@@ -141,51 +170,149 @@ export default class WarnMetricAckModal extends React.PureComponent<Props, State
+    }
+
     render() {
+        const isE0Edition = (this.props.enterpriseReady && this.props.license && this.props.license.IsLicensed === 'false');
+
         const headerTitle = (
             <FormattedMessage
                 id='warn_metric_ack_modal.header.title'
                 defaultMessage='Scaling with Mattermost'
             />
         );
-        const descriptionText = (
-            <FormattedMessage
-                id='warn_metric_ack_modal.number_of_active_users.description'
-                defaultMessage='Mattermost strongly recommends that deployments of over {limit} users upgrade to Mattermost Enterprise Edition, which offers features such as user management, server clustering, and performance monitoring'
-                values={{
-                    limit: this.props.warnMetricStatus.limit,
-                }}
-            />
-        );
 
+        let descriptionText;
+        let subText;
         const learnMoreLink = 'https://mattermost.com/pl/default-admin-advisory';
-        const subText = (
-            <div
-                style={{display: 'flex', opacity: '0.56', flexWrap: 'wrap'}}
-                className='help__format-text'
-            >
-                <FormattedMessage
-                    id='warn_metric_ack_modal.number_of_active_users.subtext'
-                    defaultMessage='By clicking Acknowledge, you will be sharing your information with Mattermost Inc., to learn more about upgrading. {link}'
-                    values={{
-                        link: (
-                            <ErrorLink
-                                url={learnMoreLink}
-                                messageId={t('warn_metric_ack_modal.learn_more.link')}
-                                defaultMessage={'Learn More'}
-                            />
-                        ),
-                    }}
-                />
-            </div>
-        );
 
-        const buttonText = (
-            <FormattedMessage
-                id='warn_metric_ack_modal.contact_support'
-                defaultMessage='Acknowledge'
-            />
-        );
+        if (!isE0Edition) {
+            if (this.props.warnMetricStatus.id === WarnMetricTypes.SYSTEM_WARN_METRIC_NUMBER_OF_ACTIVE_USERS_500) {
+                descriptionText = (
+                    <FormattedMessage
+                        id='warn_metric_ack_modal.number_of_active_users.description'
+                        defaultMessage='Mattermost strongly recommends that deployments of over {limit} users upgrade to Mattermost Enterprise Edition, which offers features such as user management, server clustering, and performance monitoring'
+                        values={{
+                            limit: this.props.warnMetricStatus.limit,
+                        }}
+                    />
+                );
+                subText = (
+                    <div
+                        style={{display: 'flex', opacity: '0.56', flexWrap: 'wrap'}}
```

Comment:
I don't know about this ... I haven't seen inline styling done anywhere else in our code and I'm not sure we should do it here. Could we do styling in a .scss file?

Reply:
It's being done in quite a few places - presumably if the settings are not reused in other components. Since now these settings are reused, I moved them to a local CSSProperties structure